index.html

```diff
@@ -2,9 +2,16 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
```

Comment:
why do we need line 5 and 6 ?

Reply:
documation of material shows in order to add all fonts which matarial uses there should be added all 3 lines.